spec/board_spec.rb

```diff
@@ -191,20 +191,60 @@
   it 'returns false when an index that is not taken is entered' do
     board = Board.new
 
-    board.mark_board(6)
-    board.mark_board(7, 'O')
-    board.mark_board(8)
+    board.mark(6)
+    board.mark(7, 'O')
+    board.mark(8)
 
     expect(board.position_taken?(4)).to eq(false)
   end
 
-  it 'returns true when an index that is taken is entered' do
+  it 'returns true when an index that is entered is already marked' do
     board = Board.new
 
-    board.mark_board(6)
-    board.mark_board(7, 'O')
-    board.mark_board(8)
+    board.mark(6)
+    board.mark(7, 'O')
+    board.mark(8)
 
     expect(board.position_taken?(8)).to eq(true)
   end
+
+  describe 'full?' do
+    it 'returns true when every position on the board is has an X mark' do
+      board = Board.new
+
+      board.squares.each.with_index do |_string, index|
+        board.mark(index)
+      end
+
+      expect(board.full?).to eq(true)
+    end
+  end
+
+  describe 'full?' do
```

Comment:
You don't need multiple `describe 'full?' do` lines. You can put these three tests under one `describe 'full?' do` block.

Reply:
Yeah - the convention is that you describe the method you're testing, and then use it for different expectations. You could also use context if you want to differentiate between different behaviors within a describe block